targets/TARGET_STM/TARGET_STM32F4/TARGET_MTS_DRAGONFLY_F411RE/ONBOARD_TELIT_HE910.cpp

```diff
@@ -19,7 +19,7 @@
 #include "cellular/onboard_modem_api.h"
 #include "UARTSerial.h"
 #include "ONBOARD_TELIT_HE910.h"
-#include "CellularLog.h"
+#include "ThisThread.h"
```

Comment:
tr_debug may need CellularLog, if not coming from other includes..?

Reply:
Once this is resolved, will start CI

Reply:
Added Cellularlog to both targets.